blog-posts/enterprises-benefit-from-scoped-npm-registries.md

```diff
@@ -1,8 +1,8 @@
 ---
-path: "/blog/scoped-registry/"
```

Comment:
You can redirect the old link to the new one in the redirect.js file. This way, the old links will still work.

Reply:
cool thanks. Didn't know this. :raised_hands:

Reply:
Maybe it's not that critical for short lived blog posts? I don't see the metaverse blog post in search engines yet neither do I see the npm blog post (which is maybe another issue?). I would skip that redirect here.

Reply:
It's not critical, but it costs close to nothing to add it. I already had the blog post in my Feedly feed, so the link is already somewhere out there 😄